### ISSUE TITLE
Requesting authTokens: generate always fails because of CORS

### ISSUE THREAD

**Issue by ilovedesert001** [firebase/firebase-js-sdk#2468]
* Operating System version: mac
  * Browser version: chrome Chrome/76.0.3800.0
  * Firebase SDK version: 7.6.1
  * Firebase Product:  analytics , prformance

x-firebase-client: fire-core/0.5.0 fire-js/ firebase-app/7.6.1 fire-iid/0.3.8 fire-analytics/0.2.9 fire-perf/0.2.28

sdkVersion: "w:0.3.8"

```
Access to fetch at 'https://firebaseinstallations.googleapis.com/v1/projects/xxxxxxxxxxxxxxxxx/authTokens:generate' from origin 'https://www.xxxxxx.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by mmermerkaya** [comment]
Duplicate of #2467
